getting-started/modules/zgw/notificaties-api-authentication.md

```diff
@@ -0,0 +1,52 @@
+# Notificaties API Authentication
```

Comment:
I think this page needs to be referred to in at least `SUMMARY.md`, otherwise it's not accessible from anywhere.